README.md

```diff
@@ -133,6 +135,44 @@ will fade away and you will mostly focus on the code within the braces.
 
 **Note**: Annotations are available via the **[mobx_codegen](https://github.com/mobxjs/mobx.dart/tree/master/mobx_codegen)** package.
 
+### Readonly
```

Comment:
all of this can be move to the `observables.mdx`

Reply:
I have already moved the previous README to `observables.mdx`. Should I just remove the readonly from the README?

Reply:
Ya let's remove from readme

Reply:
Done